static/sass/_pattern_lists.scss

```diff
@@ -375,3 +376,31 @@
     }
   }
 }
+
+@mixin ubuntu-p-faq-stepped-list--detailed {
+  .p-faq-stepped-list--detailed {
+    @extend .p-stepped-list--detailed;
+
+    .p-faq-stepped-list__item {
+      @extend .p-stepped-list__item;
+
+      padding-top: 0;
+    }
+
+    @supports (display: grid) {
+      .p-faq-stepped-list__content {
+        @media (min-width: $threshold-6-12-col) {
+          display: flex;
+          grid-column-end: span 12;
+          margin-top: 0;
+        }
+      }
+    }
+
+    // TODO : the accordion component needs to be fixed in vanilla. The issue was filed on https://github.com/canonical/vanilla-framework/issues/4563. This styles written locally needs to be removed after vanilla update.
```

Comment:
There is a mix of `XXX` and `TODO` in this branch. XXX is the standard to indicate a to-do.